examples/dbscan/ArborX_DBSCAN.hpp

```diff
@@ -52,15 +52,23 @@ namespace DBSCAN
 {
 
 template <typename MemorySpace>
-struct NumNeighCallback
+struct NumNeighEarlyExitCallback
```

Comment:
Nitpick: no need to rename the callback.

Reply:
I just thought that if the original name is retained, I would expect the result to be the exact number of neighbors, which is no longer true.